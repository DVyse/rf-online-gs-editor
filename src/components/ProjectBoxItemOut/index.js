/**
 *
 * ProjectBoxItemOut
 *
 */

import React from 'react';
// import PropTypes from 'prop-types';
// import styled from 'styled-components';

function ProjectBoxItemOut() {
  return <div />;
}

ProjectBoxItemOut.propTypes = {};

export default ProjectBoxItemOut;
