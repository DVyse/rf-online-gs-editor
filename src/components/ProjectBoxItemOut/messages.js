/*
 * ProjectBoxItemOut Messages
 *
 * This contains all the text for the ProjectBoxItemOut component.
 */

import { defineMessages } from 'react-intl';

export default defineMessages({
  Disable: {
    id: 'app.components.ProjectBoxItemOut.Disable',
    defaultMessage: 'Disable',
  },
  TotalProbability: {
    id: 'app.components.ProjectBoxItemOut.TotalProbability',
    defaultMessage: 'Total probability of falling out of objects',
  },
  isCorrect: {
    id: 'app.components.ProjectBoxItemOut.isCorrect',
    defaultMessage: 'is correct',
  },
  incorrect: {
    id: 'app.components.ProjectBoxItemOut.incorrect',
    defaultMessage: 'incorrect',
  },
  SelectItem: {
    id: 'app.components.ProjectBoxItemOut.SelectItem',
    defaultMessage: 'Select item',
  },
});
