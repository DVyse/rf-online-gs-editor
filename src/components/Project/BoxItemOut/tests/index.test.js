import React from 'react';
import { shallow } from 'enzyme';

import ProjectBoxItemOut from '../index';

describe('<ProjectBoxItemOut />', () => {
  it('Should exist', () => {
    const wrapper = shallow(<ProjectBoxItemOut />);
    expect(wrapper).toMatchSnapshot();
  });
});
