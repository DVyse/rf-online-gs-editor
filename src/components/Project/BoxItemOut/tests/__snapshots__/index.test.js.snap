// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProjectBoxItemOut /> Should exist 1`] = `<div />`;
